docs/index.rst

```diff
@@ -36,6 +36,8 @@ Day-to-Day Usage
 ================
 
 - Once you're comfortable with the concepts, our :doc:`api` contains all information you need to use ``attrs`` to its fullest.
+- If you want to put objects into sets or use them as keys in dictionaries, they have to be hashable.
+  Unfortunately the topic is more complex than it seems but :doc:`hashing` will give you a primer on what to look out for.
```

Comment:
Suggestion: "... they have to be hashable. The simplest way to do that is to use `frozen=True`. Unfortunately ..."